### PR TITLE
Tool version update

### PR DESCRIPTION
**Pull request by snadampal**

### CODE REVIEW

**Comment by nSircombe** [comment]
Thanks for the PR @snadampal. 
It looks good to me, but I'll just run it through our downstream testing before merging.

File: docker/pytorch-aarch64/Dockerfile [hunk 1]
```diff
@@ -233,7 +233,7 @@ COPY patches/site.cfg $PACKAGE_DIR/site.cfg
 RUN $PACKAGE_DIR/build-numpy.sh
 
 # Install some  basic python packages needed for SciPy
-RUN pip install --no-cache-dir pybind11==2.6.2 pyangbind
+RUN pip install --no-cache-dir pybind11==2.6.2 pyangbind pythran
```
Comment (nSircombe) [inline docker/pytorch-aarch64/Dockerfile:236]: Just curious to know why pythran is needed?

File: docker/pytorch-aarch64/Dockerfile [hunk 2]
```diff
@@ -203,8 +203,8 @@ ENV PY_VERSION="${default_py_version}" \
     CPU="${cpu}" \
     ARCH="${arch}"
 # Key version numbers
-ENV NUMPY_VERSION=1.19.5 \
-    SCIPY_VERSION=1.5.2 \
+ENV NUMPY_VERSION=1.22.1 \
+    SCIPY_VERSION=1.7.3 \
```
Comment (nSircombe) [inline docker/pytorch-aarch64/Dockerfile:207]: This combination of NumPy and SciPy don't appear to be compatible. Running the simple examples from the finished image (https://github.com/ARM-software/Tool-Solutions/blob/master/docker/pytorch-aarch64/examples/README.md) generates an error:

```
UserWarning: Failed to initialize NumPy: module compiled against API version 0xf but this version of numpy is 0xe (Triggered internally at  ../torch/csrc/utils/tensor_numpy.cpp:68.)
```

Rolling back the NumPy version to 1.21.5 appears to fix the issue.

File: docker/tensorflow-aarch64/Dockerfile [hunk 3]
```diff
@@ -191,8 +191,8 @@ ENV PY_VERSION="${default_py_version}" \
     CPU="${cpu}" \
     ARCH="${arch}"
 # Key version numbers
-ENV NUMPY_VERSION=1.19.5 \
-    SCIPY_VERSION=1.5.2 \
+ENV NUMPY_VERSION=1.22.1 \
```
Comment (nSircombe) [inline docker/tensorflow-aarch64/Dockerfile:194]: I didn't encounter any problems using TensorFlow with these version numbers, but I think that's simply that the tests & examples don't have a SciPy dependency.
In any case, it is preferable to keep the version numbers across the TensorFlow and Torch builds consistent, so I would suggest opting for NumPy 1.21.5 here too,

**Review by nSircombe — CHANGES_REQUESTED** [review]
I believe the NumPy version will have to be rolled back slightly to 1.21.5 to avoid an API compatibility issue with SciPy 1.7.3.

The python examples provided in the `examples` folder should provide a simple test that the updated build is working as expected.

File: docker/pytorch-aarch64/Dockerfile (hunk 1 above)
Comment (nSircombe) [reply docker/pytorch-aarch64/Dockerfile:236]: ...no matter, I've found it - new SciPy dependency.

File: docker/pytorch-aarch64/Dockerfile (hunk 2 above)
Comment (snadampal) [reply docker/pytorch-aarch64/Dockerfile:207]: @nSircombe : Thanks for the feedback and suggestions! I will update the PR with the fix.

File: docker/tensorflow-aarch64/Dockerfile (hunk 3 above)
Comment (snadampal) [reply docker/tensorflow-aarch64/Dockerfile:194]: I agree with having the same version across pytorch and tensorflow.

File: docker/pytorch-aarch64/Dockerfile (hunk 2 above)
Comment (snadampal) [reply docker/pytorch-aarch64/Dockerfile:207]: @nSircombe the issue is with pytorch docker build downgrading the 'numpy' version from 1.22.1 to 1.21.5 during 'numba-0.5.0' installation, because it has dependency on NumPy >=1.18,<1.22.

Here is the root cause for the torch failure:
1. All the torch libraries have been built with numpy1.22.1
2. while installing the dependent packages for benchmarking (namba via librosa), numpy version is changed from 1.22.1 to 1.21.5. thats why torch libraries built with 1.22.1 version are no longer usable.

One option is to limit numy version <1.22, the next lower version is 1.21.5.
Or Remove librosa from the docker.-->  The examples are still passing when removed librosa. Any details on which exact benchmark is dependent on it.

I'm fine with limiting numpy version to 1.21.5 (this already has the latest OpenBLAS version with the aarch64 fixes), but we will be missing any critical fixes from numpy later versions, hence checking on option to remove librosa.

Comment (nSircombe) [reply docker/pytorch-aarch64/Dockerfile:207]: Hi @snadampal,

Thanks for doing a bit more digging. It looks like the librosa (and, therefore the numba dependency, and unwanted downgrade of bumpy you've found) is used for the MLCommons RNNT example described here: https://github.com/ARM-software/Tool-Solutions/tree/master/docker/pytorch-aarch64/examples#rnnt, which we run as part of our testing with `--accuracy` set.
It looks like the dependency is introduced here: https://github.com/mlcommons/inference/blob/2ef16835ac66b3ee03e1fc7fe6f95ec9689200d3/speech_recognition/rnnt/pytorch/parts/segment.py#L16

Comment (snadampal) [reply docker/pytorch-aarch64/Dockerfile:207]: Thanks. Then aligned on downgrading the numpy version to 1.21.5 for both pytorch as tensorflow builds.

**Review by nSircombe — APPROVED** [review]
Looks good to me, thanks!
